triage/roster_log_review_queue/extract_pack.py

```
@@ -0,0 +1,136 @@
+    for name in z.namelist():
+        if name.lower().endswith(".xlsx"):
+            return z.read(name)
+    raise FileNotFoundError("No .xlsx in reference ZIP")
+
+
+def extract_operator_cf_pack(xlsx_bytes: bytes) -> dict:
+    with zipfile.ZipFile(io.BytesIO(xlsx_bytes), "r") as z:
+        styles = read_text(z, "xl/styles.xml")
+        dxf_styles = _extract_dxf_list(styles)
+        nm = sheet_name_map(z)
+        part = [p for p, n in nm.items() if n == "Live - June 2026"][0]
```

Ruff (0.15.15)

[warning] Prefer `next(...)` over single element slice

Replace with `next(...)`

(RUF015)